src/components/SingleMovie.js

```diff
@@ -1,15 +1,31 @@
 import React from "react"
+import "../css/SingleMovie.css"
 
 function SingleMovie ({movieInFocus, movies}) {
 
   const correctMovie = movies.find(movie => movie.id === movieInFocus)
 
   return (
-    <div>
-      <img src={correctMovie.poster_path} height="300" width="200"></img>
-      <p>Title: {correctMovie.title}</p>
-      <p>Rating: {correctMovie.average_rating}</p>
-      <p>Release Date {correctMovie.release_date}</p>
+    <div className="SingleMovie" style={{
+      backgroundImage:`linear-gradient(to bottom, transparent 0%, black 60%),
+      url(${correctMovie.backdrop_path})`,
+      backgroundPosition: 'center',
+      backgroundSize: 'cover',
+      backgroundRepeat: 'no-repeat',
+      width:'100vw',
+      height:'100vh'
+      }}>
+      <h3 className="single-movie-title"> 
+        {correctMovie.title}
+        <h4 className="single-movie-tagline"> {correctMovie.tagline}</h4>
```

Comment:
A h4 is nested inside of an h3 right here and dev tools is throwing errors, if you move line 20 down one line, all should be well.